docs/index.html

```diff
@@ -7,56 +7,49 @@
 <div class="panel panel-info">
     <div class="panel-heading">
         {% include icon.html type="info" %}
-        ev3dev is a community-maintained project that relies on people's contributions.
+        ev3dev is a community-maintained project that relies on community contributions.
     </div>
     <div class="panel-body">
         <p>
-            If there's a
-            topic that you think we should have docs for, please consider
-            <a href="../contribute">contributing</a>.
+            If there's a topic that you think we should have docs for, please
+            consider <a href="../contribute">contributing</a>.
         </p>
         <p>
             <a href="https://github.com/ev3dev/ev3dev/issues/153#issuecomment-93602107">Some pages</a>
-            have not been migrated from our old wiki yet. We'd love some help with getting these last few pages migrated:
-            if you would like to help out, read more about what's needed in the <a href="../contribute#write-some-documentation">documentation section</a>
+            have not been migrated from our old wiki yet. We'd love some help
```

Comment:
I like the idea of some variety of reward, but I think it opens up some problems; namely, "Who supplies the reward?" and "How do we determine if someone gets it?"

I might be over-thinking this, though. We could probably do something simple and cheap with rewards like that and get reasonable results. Or maybe Bountysource does something to deal with this problem. We'd just need to make sure we were being fair and that we don't get into grey area if someone contributes lackluster content.

Either way, I agree that we should do _something_. Having this alert here is less than ideal if it isn't serving a useful purpose. _Although, I have to say, it does visually break up the page a bit._